src/apis/staked-relayer.ts

```diff
@@ -5,9 +5,11 @@ import { ApiPromise } from "@polkadot/api";
 import { VaultsAPI, DefaultVaultsAPI } from "./vaults";
 import BN from "bn.js";
 import { pagedIterator } from "../utils";
+import StorageKey from "@polkadot/types/primitive/StorageKey";
 
 export interface StakedRelayerAPI {
     list(): Promise<ActiveStakedRelayer[]>;
+    listIncludingIds(): Promise<[StorageKey, ActiveStakedRelayer][]>;
```

Comment:
We would want the AccountId instead of the StorageKey. Also, can you please return that as a `Map` similar to the issue/redeem requests?